src/bundle.js

```diff
@@ -31,13 +32,14 @@ export const bundleToFile = (fName, outputFile, force) => {
     },
   };
 
-  const ctx = createContext(document, doc, resolvedFileName, lintConfig);
+  const redoclyClient = new RedoclyClient();
```

Comment:
Why we need passing `redoclyClient` to `createContext`? Can't `createContext` create it itself?

Reply:
agree, that makes sense